src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,24 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        /// <summary>
+        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
+        /// </summary>
+        /// <returns>A string representation of the current exception.</returns>
+        public override string ToString()
+        {
+            StringBuilder text = new StringBuilder();
+            text.Append(SR.ReflectionTypeLoad_LoadFailed.ToString());
```

Comment:
Is the `.ToString()` necessary?

Reply:
It addresses the concern from https://github.com/dotnet/corefx/issues/25906

Reply:
Maybe I'm missing something...

Instead of:

```c#
text.AppendLine(SR.ReflectionTypeLoad_LoadFailed.ToString());
```

Is there any reason it cannot be:

```c#
text.AppendLine(SR.ReflectionTypeLoad_LoadFailed);
```